eng/Versions.props

```diff
@@ -23,6 +23,9 @@
     <MicrosoftBclAsyncInterfacesVersion>1.0.0-preview7.19311.1</MicrosoftBclAsyncInterfacesVersion>
     <!-- core-setup -->
     <MicrosoftNETCoreAppVersion>3.0.0-preview7-27820-07</MicrosoftNETCoreAppVersion>
+    <MicrosoftNETCoreDotNetHostPackageVersion>3.0.0-preview7-27820-07</MicrosoftNETCoreDotNetHostPackageVersion>
```

Comment:
These will be always the same as MicrosoftNETCoreAppVersion. Is it useful to have separate properties for each of them?

If these properties are useful, I think each of them will need subscriptions in Version.Details.xml to get auto-updated.

Reply:
Thanks; I'll simplify to just MicrosoftNETCoreAppVersion. Note that corefx\eng\Versions.props does have (most) of these; I was mimicking that.